community.hachyderm.io/content/en/docs/moderation/_index.md

```diff
@@ -0,0 +1,358 @@
+---
+title: "Moderation Guiding Principles"
+linkTitle: "Moderation Guiding Principles"
+weight: 4
+description: >
+  Hachyderm Moderation: Process and Responsibilities
+---
+
+This page includes:
+
+* Information about how we moderate Hachyderm.
+* The responsibilities of members (Hachydermians) and moderators
+(the "we/us" of this document).
+* Other supporting context relevant to the moderation process.
+
+We appreciate that this is probably not the page that you are
+coming to for light reading. Most likely you are here because you
+have been moderated and are trying to figure out what to do
+or you are a prospective user and you are trying to figure out if
+this instance is a safe home for you. We are here for you and have
+tried to build as comprehensive and clear a document as possible.
+
+For full transparency: we are currently designing our
+documentation architecture, so it is likely
+that this singular document will be split onto different pages for
+clarity. Please keep this in mind if you are bookmarking this
+page.
+
+(Skip to the [Help, I was moderated!](#help-i-was-moderated)
+section.)
+
+# Responsibilities 
+
+## Hachydermians
+
+**Hachydermians are responsible for their own actions.** This means:
+
+1. Each Hachydermian is responsible for following server rules
+(and relevant laws).
+1. Each Hachydermian is responsible for ensuring that their
+participation in the fediverse is collaboratively aligned.
+1. Each Hachydermian acknowledges that it is our collective
+responsibility to create and steward the interactions we want to
+see.
+1. Each Hachydermian is responsible for their own learning and growth.
+
+## Moderators (of Hachyderm)
+
+**Moderators are responsible for stewarding safe spaces by
+enforcing good conduct.** This means:
+
+1. "Good conduct" here means "following server rules and local
```

Comment:
I don't think we can or should put the onus on moderators as stewards of local laws.  there's too much knowledge required.  if it is the responsibility of the user to abide by those laws then that is sufficient.